homestar-runtime/tests/cli.rs

```diff
@@ -309,3 +324,159 @@ fn test_daemon_serial() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+#[serial]
+#[cfg(windows)]
+fn test_windows_signal_kill() -> Result<()> {
```

Comment:
```suggestion
fn test_signal_kill_serial()
```

Don't need windows as the cfg is explicit.